### DISCUSSION TITLE
3D viewport background is black

### DISCUSSION THREAD

**Discussion by Thomasnerd06** [Ifsttar/I-Simpa#350] [Q&A] [answered]
I've just loaded up I-simpa for the first time, and the viewport is black, which makes selecting anything difficult, as I have no clue what is selected.
Shown here:
![black viewport problem](https://user-images.githubusercontent.com/54401430/213921528-fb5e70d0-4a95-4f8b-9ebd-0d093dd2c2fa.png)

Thanks in advance,
Tom.

**Comment by nicolas-f** [comment, marked as answer]
Hi,

Yes there is an issue at the first startup. You have to close I-Simpa and restart it again.

Il will be fixed when we will switch to a real 3d engine.

Thx

**Reply by Thomasnerd06** [reply]
Hi, this was sort of the solution. It didnt fix with a restart of the program, though restarting the machine fixed the issue
